pkg/combustion/templates/15-rke2-multi-node-installer.sh.tpl

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+declare -A hosts
+
+{{- range .Nodes }}
+hosts[{{ .Hostname }}]={{ .Type }}
+{{- end }}
+
+HOSTNAME=$(cat /etc/hostname)
+if [ ! "$HOSTNAME" ]; then
+    echo "ERROR: Could not identify whether the host is an RKE2 server or agent due to missing hostname"
+    exit 1
+fi
```

Comment:
Not sure I've tested this, but isn't `/etc/hostname` by default `localhost.localdomain` ?

Reply:
It will default to `localhost.localdomain` but at combustion time the value is an empty string for some reason. You can verify it by configuring multi node cluster and using a NIC not known by nmc.

Reply:
No worries. So long as this check works, I think we're good.